packages/core/base/src/wallet.ts

```diff
@@ -13,7 +13,30 @@ import type { IdentifierArray, IdentifierRecord, IdentifierString } from './iden
  *
  * @group Wallet
  */
-export type WalletVersion = '1.0.0';
+export type WalletVersion = WalletVersion1_0_0 | WalletVersion1_1_0;
+
+/**
+ * Initial version of the Wallet Standard.
+ *
+ * @group Wallet
+ */
+export type WalletVersion1_0_0 = '1.0.0';
+
+/**
+ * Version of the Wallet Standard that supports HTTP/S URLs for icons.
+ *
+ * @group Wallet
+ */
+export type WalletVersion1_1_0 = '1.1.0';
+
+/**
+ * Icon of a {@link Wallet} or {@link WalletAccount}.
+ *
+ * @group Wallet
+ */
+export type WalletIcon<V extends WalletVersion = WalletVersion1_0_0> = V extends WalletVersion1_1_0
+    ? WalletIcon1_1_0
+    : WalletIcon1_0_0;
```

Comment:
Maybe the other way around, so that when we go to `>1.1.0` this doesn't have to change?
```suggestion
export type WalletIcon<V extends WalletVersion = WalletVersion1_0_0> = V extends WalletVersion1_0_0
    ? WalletIcon1_0_0
    : WalletIcon1_1_0;
```

Reply:
I was thinking that if there's no value for the generic type provided, it should default to the version wallets/apps use now, so that it constitutes a nonbreaking change?

Reply:
How

Reply:
That will still happen! Currently your code does this:

* No type param -> Icon 1.0
* Wallet 1.0 type param -> Icon 1.0
* Wallet 1.1 type param -> Icon 1.1
* Wallet 1.2 type param -> Icon 1.0

I'm saying if you switch it, then it will do this:

* No type param -> Icon 1.0
* Wallet 1.0 type param -> Icon 1.0
* Wallet 1.1 type param -> Icon 1.1
* Wallet 1.2 type param -> Icon 1.1